lib/FieldType/DataTransformer/CheckboxValueTransformer.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryForms\FieldType\DataTransformer;
+
+use eZ\Publish\Core\FieldType\Checkbox\Value;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class CheckboxValueTransformer implements DataTransformerInterface
```

Comment:
@bdunogier I'm agree. I think we can even replaces those transformers with EzSystems\RepositoryForms\FieldType\DataTransformer\FieldValueTransformer, behind the scene they are doing that same thing (I've checked and it works fine).

I assume this changes should be pushed in two separate commits: 1st  with implementation for checkbox, integer, float and isbn content field types, and 2nd with replacment of TextLineValueTransformer  in favor FieldValueTransformer in TextLineFormMapper. Am I right ?